internal/repositories/devicedefinition/devicedefinition.go

```diff
@@ -0,0 +1,224 @@
+package devicedefinition
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	gmodel "github.com/DIMO-Network/identity-api/graph/model"
+	"github.com/DIMO-Network/identity-api/internal/helpers"
+	"github.com/DIMO-Network/identity-api/internal/repositories/base"
+	"github.com/vektah/gqlparser/v2/gqlerror"
+	"golang.org/x/exp/slices"
+	"net/http"
+	"net/url"
+	"path"
+	"strings"
+)
+
+// TokenPrefix is the prefix for a global token id for Device Definition.
+const TokenPrefix = "DD"
+
+type DeviceDefinitionRepository struct {
+	*base.Repository
+}
+
+func ToAPI(v *DeviceDefinitionTablelandModel) *gmodel.DeviceDefinition {
+	var result = gmodel.DeviceDefinition{}
+
+	result.ID = v.ID
```

Comment:
These IDs do not have the same meaning as the model.DeviceDefinition.ID is an ID specific to the identity-API that allows users to look up the Query for the DeviceDefinition using the Node query 
as seen here https://github.com/DIMO-Network/identity-api/blob/4aac3c1fedec102555242b93195aeba37fca5b05/graph/schema.resolvers.go#L23-L43